modules/ebi-metagenomics/bmtagger/index_reference/main.nf

```diff
@@ -0,0 +1,32 @@
+process CREATE_DB_BMTAGGER {
+
+    label 'process_low'
+
+    conda "bioconda::bmtagger==3.101"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bmtagger:3.101--h470a237_4':
+        'biocontainers/bmtagger:3.101--h470a237_4' }"
+
+    input:
+    path(reference_fasta)
+
+    output:
+    path("${reference_fasta.baseName}.bitmask")         , emit: bitmask
+    path("${reference_fasta.baseName}.srprism.*")       , emit: srprism
+    path("${reference_fasta}.n*")              , emit: blast_db
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
```

Comment:
I think we should include the args too.. for example to tweak the parameters of the different tools (such as -A and -w"